examples/appsync/apikey.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: appsync.aws.upbound.io/v1beta1
+kind: APIKey
+metadata:
+  annotations:
+    meta.upbound.io/example-id: appsync/v1beta1/apikey
+  labels:
+    testing.upbound.io/example-name: example
+  name: example
+spec:
+  forProvider:
+    apiIdSelector:
+      matchLabels:
+        testing.upbound.io/example-name: example
+    region: us-west-1
+---
+apiVersion: appsync.aws.upbound.io/v1beta1
+kind: GraphQLAPI
+metadata:
+  annotations:
+    meta.upbound.io/example-id: appsync/v1beta1/apikey
```

Comment:
@turkenf, please add a pre-delete hook here that deletes APIKey first to ensure `GraphQLAPI` is not deleted before  `APIKey`. This seems to be root cause of the current uptest failure.